odoo/addons/base/models/res_partner.py

```diff
@@ -485,6 +492,10 @@ def _clean_website(self, website):
 
     @api.multi
     def write(self, vals):
+        if vals.get('active') == False:
```

Comment:
Shouldn't we use "is" in python ?

Reply:
I tested it and the result is exactly the same. Do you think it's clearer ?

Reply:
It's just as concise, but slightly more correct:
```python
>>> 0 == False
True
>>> 0 is False
False
```
In practice, not likely to make a difference here, just FYI.